controllers/reconcile.go

```diff
@@ -1760,16 +1760,21 @@ func (r *FrontendReconciliation) createBaseWidgetDashboardTemplatesConfigMap(nn
 
 	cfgMap = r.setupConfigMapWithLabels(nn, markForRestart)
 
-	baseWidgetDashboardTemplates := setupBaseWidgetDashboardTemplates(frontendList)
+	if sourceConfigMap != nil {
+		r.Log.Info("Using data from existing config map", "targetConfigMapName", cfgMap.Name, "targetConfigMapNamespace", cfgMap.Namespace, "sourceConfigMapName", sourceConfigMap.Name, "sourceConfigMapNamespace", sourceConfigMap.Namespace)
+		cfgMap.Data = sourceConfigMap.Data
```

Comment:
@Hyperkid123 FWIW this will obviously cause the generated config maps to point to the same underlying map versus a true clone. I thought that was ok as I can't see a reason for the config maps to differ in the various target namespaces (at least in the short term). Let me know if you disagree.

Reply:
Yeah that is ok. I'd also like to rebase and merge the tests draft PR i've opened yesterday. Just to make sure we don't have this happening later on.

Reply:
I'd rather see the config map data as a return value of the function and then another function that creates the config maps in k8s, but that can be adressed in some nice refactoring round. The fact the issue is fixed now is the most important.